apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -130,6 +130,13 @@
                     "type" : "boolean",
                     "default": true
                 },
+                "gossiped_peers" : {
+                    "description" : "The number of peers sent in ping message",
+                    "type" : "integer",
+                    "minimum" : 1,
+                    "maximum" : 32,
```

Comment:
is this maximum really enforced ?

Reply:
it should be by this config schema?

Reply:
Good question, I started at the code and could not see where the `share` value is compared to the incoming ping value... I'll look more carefully again. Not a blocker, but we should understand this.

Reply:
#3086